components/automate-ui/src/app/components/form-field/form-field.component.ts

```diff
@@ -15,9 +20,31 @@ import { ErrorDirective } from '../error/error.directive';
               '../input/input.directive.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class FormFieldComponent implements AfterContentInit {
+export class FormFieldComponent implements AfterContentInit, OnChanges {
   @ContentChildren(ErrorDirective) errorChildren: QueryList<ErrorDirective>;
 
+  // A FormFieldComponent can optionally track whether an input field (FormControl)
+  // being edited is the same as, or different from, the field's initial value.
+  // That is, it knows whether the user has made a change or not,
+  // and marks the pristine/dirty state accordingly, so that information may be used
+  // in the parent form to e.g., disable the `Save` button, per our UX guidelines.
+  // Example:
+  // <form [formGroup]="updateForm">
+  //   <chef-form-field
+  //       [watch]="updateForm.controls.name.value"
+  //       [control]="updateForm.controls.name">
+  //     <label>Name <span aria-hidden="true">*</span></label>
+  //     <chef-input formControlName="name" type="text"></chef-input>
+  //   </chef-form-field>
+  // </form>
+  // <chef-button primary [disabled]="!updateForm.valid || !updateForm.dirty">
+  //
+  // You need to supply these two parameters:
+  @Input() watch: string; //  dynamic value to watch
```

Comment:
❓ every thing we watch is a `string`? Other things can't happen... I guess so... chef-phat-radio is a string in the end, too, I guess... Do we have numbers?